rules/windows/process_creation/proc_creation_win_office_outlook_susp_child_processes_remote.yml

```
@@ -0,0 +1,29 @@
+title: Suspicious Outlook Remote Child Process
```

```suggestion
title: Suspicious Remote Child Process from Outlook 
```
At first reading I thought it was outlook the remote process.